core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5928,7 +5972,13 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WEB_FILE_INFO_ENABLED = "alluxio.web.file.info.enabled";
     public static final String WEB_RESOURCES = "alluxio.web.resources";
     public static final String WEB_THREADS = "alluxio.web.threads";
+    public static final String WEB_CORS_ALLOW_CREDENTIAL = "alluxio.web.cors.allow.credential";
+    public static final String WEB_CORS_ALLOW_HEADERS = "alluxio.web.cors.allow.headers";
+    public static final String WEB_CORS_ALLOW_METHODS = "alluxio.web.cors.allow.methods";
+    public static final String WEB_CORS_ALLOW_ORIGINS = "alluxio.web.cors.allow.origins";
```

Comment:
These three "allow" should be changed to "allowed"? WDYT @tieujason330

Reply:
i'd say `allow` to keep it consistent with HTTP headers (e.g. `Access-Control-Allow-Origin`)